src/main/webapp/app/guided-tour/tours/tutor-assessment-tour.ts

```diff
@@ -14,7 +14,7 @@ export const tutorAssessmentTour: GuidedTour = {
             highlightPadding: 10,
             orientation: Orientation.LEFT,
             permission: ['ROLE_TA'],
-            pageUrl: '/course',
+            pageUrl: '/course-management',
```

Comment:
@lovaiible I did update this Tour, not sure if this is still correct. Can you verify this?

Reply:
@jpbernius Thank you for updating the tour. I tested it and all steps worked for me.